gemfiles/activerecord_6.gemfile

```diff
@@ -0,0 +1,7 @@
+# This file was generated by Appraisal
```

Comment:
@tpendragon Was this expected to be used?

Reply:
Good catch! Removed.